modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JaxRSServerCodegen.java

```diff
@@ -167,6 +167,15 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
     }
 
     @Override
+    public String toApiName(String name) {
+        if (name.length() == 0) {
+            return "DefaultApi";
+        }
+        name = name.replaceAll("[^a-zA-Z ]+", "_");
```

Comment:
i copied that line from [here](https://github.com/xhh/swagger-codegen/blob/jax-rs-classname/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java#L389) which is used to generate the `classname` in other languages
but i agree with you, let me update it to allow numbers